### DISCUSSION TITLE
How to identify strings affected by changes in the terminology after the glossary has been updated?

### DISCUSSION THREAD

**Discussion by Akemi-Madoka** [WeblateOrg/weblate#11622] [Q&A]
In our project, we have maintained a glossary. 
However, after discussion, we decided to modify the translation of one or several terms. 
This raises an issue: how do we identify those strings that used the previous versions of the term translations?

I attempted to implement "check-glossary," but the result was completely different from what I expected.
 It seems that strings using terminology from previous versions did not trigger the relevant checks, 
 and even when I manually altered them to provoke a trigger, it still failed to launch.

Could it be due to performance issues that the related checks are conducted asynchronously and with a delay? However, I haven't noticed any queue...

**Comment by nijel** [comment]
The glossary check needs to be turned on and the checks are then executed in the background.

PS: See https://docs.weblate.org/en/latest/user/checks.html#does-not-follow-glossary
